vector/src/main/java/im/vector/app/features/crypto/verification/choose/VerificationChooseMethodFragment.kt

```diff
@@ -100,6 +100,18 @@ class VerificationChooseMethodFragment @Inject constructor(
         sharedViewModel.itWasNotMe()
     }
 
+    override fun acceptRequest() {
+        withState(viewModel) {
```

Comment:
Why do you need the state here?

Reply:
Used to because was passing data, but now it's an object, will update